src/benchmarks/pmembench.c

```diff
@@ -628,7 +608,7 @@ pmembench_get_total_results(struct latency *stats, double *workers_times,
 	size_t nresults = repeats * nworkers;
 	size_t i;
 	size_t j;
-	size_t d;
+	double d;
```

Comment:
There's probably a reason this is size_t. There's even a second variable "df" which I assume stands for "d float".